examples/si7021_simpletest.py

```diff
@@ -1,10 +1,12 @@
+"""
+Initializes the sensor, gets and prints readings every two seconds.
+"""
 import time
 import board
-import busio
 import adafruit_si7021
 
 # Create library object using our Bus I2C port
-i2c = busio.I2C(board.SCL, board.SDA)
+i2c = board.I2C()
 sensor = adafruit_si7021.SI7021(i2c)
```

Comment:
You could just do `sensor = adafruit_si7021.SI7021(board.I2C())`. I am not sure whether you'd think that's clearer or not.
`

Reply:
Is it possible to use `board.I2C()` multiple times? I tend to keep it in a variable like this for when I add a second device into the mix. Though come to think of it I'm not sure I ever actually tried calling it twice instead of storing it in a variable.

This being a simpletest though I do see the appeal of putting it inline like that. I'll commit  that change in a moment.

Reply:
Yes, it returns the singleton. So the first time it's called, it creates the object, and then after that it returns the already-created object.